ontology/yaml/resources/ELECTRICAL/entity_types/GEN.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GEN_SS:
+  description: "Standard Generator unit."
+  is_canonical: true
+  implements:
+  - GEN
+  - SS  
+
+GEN_RMM:
```

Comment:
why is this not combined with all the other types? these feel like they all need to be combined. the types seem incomplete.

Reply:
The independent canonical have been put together to create single canonical.
GEN_SS_RMM_TKLS_OTPM_PWM_RDM